src/components/AppDownloadBadges/AppDownloadBadges.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="app-download-badges">
+    <a
+      class="phone:hidden"
+      :href="url"
+      target="_blank"
+    ><Apple /></a>
+    <a
+      class="phone:hidden"
+      :href="url"
+      target="_blank"
+    ><Google /></a>
+    <a
+      class="button rounded-8 text-like-green text-14 text-center font-400 m-8 p-12 phone:inline tablet:hidden laptop:hidden"
+      :href="url"
+      target="_blank"
+    >{{ $t(`AppDownloadBadges.${type === 'start' ? 'Start' : 'Download'}`) }}</a>
+    <a
+      v-if="type === 'start'"
+      class="text-12 text-like-green hover:underline phone:inline tablet:hidden laptop:hidden"
+      :href="url"
+      target="_blank"
+    >{{ $t('AppDownloadBadges.Download') }}</a>
+  </div>
+</template>
+
+<script>
+import Apple from './apple.svg';
+import Google from './google.svg';
+
+export default {
+  name: 'AppDownloadBadges',
+  components: {
+    Apple,
+    Google,
+  },
+  props: {
+    type: {
+      type: String,
+      default: 'default',
+    },
+    from: {
+      type: String,
+      default: '',
+    },
+  },
+  computed: {
+    url() {
+      let url = 'https://likerland.app.link/';
```

Comment:
should `'https://likecoin.page.link/likeco'` from `getAppURL`  be used if no `from` exists to unify analytics behaviour?